controllers/reconcile.go

```diff
@@ -951,6 +951,17 @@ func setupFedModules(feEnv *crd.FrontendEnvironment, frontendList *crd.FrontendL
 				module.FullProfile = crd.TruePtr()
 			}
 
+			if frontend.Spec.Frontend.Paths != nil && len(frontend.Spec.Frontend.Paths) > 0 {
+				module.CDNPath = frontend.Spec.Frontend.Paths[0]
+				// make sure the path start and ends with "/"
+				if !strings.HasPrefix(module.CDNPath, "/") {
```

Comment:
should these checks be outside of the `if`? i.e. regardless if we copy `frontend.Spec.Frontend.Paths[0]` to `module.CDNPath`  ?